synapse/rest/key/v2/local_key_resource.py

```diff
@@ -49,7 +47,8 @@ class LocalKey(Resource):
                     "key": # base64 encoded NACL verification key.
                 }
             }
-            "tls_certificate": # base64 ASN.1 DER encoded X.509 tls cert.
+            "tls_fingerprints": # Fingerprints of the TLS certs this server uses.
+                - {"sha256": "..."}
```

Comment:
1. This is JSON not yaml
2. Are we actually changing the API here? Won't that affect when other servers look up keys?